tests/ansible_collections/roles/remove-rcmtools/tasks/clean_up_el8.yml

```
@@ -0,0 +1,5 @@
+---
+- name: Remove rcmtools packages installed on the system
+  ansible.builtin.shell: dnf list --installed | grep @rcmtools | awk '{print $1}' | xargs dnf remove -y
+- name: Remove rcmtools repo from the system
+  ansible.builtin.shell: rm -f /etc/yum.repos.d/rcmtools.repo
```

```suggestion
  ansible.builtin.yum_repository:
    file: rcmtools
    name: rcmtools
    state: absent

- name: Clean yum metadata
  ansible.builtin.command: yum clean metadata
```
Same as in clean_up_el8.yml

---

Done